core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -43,6 +43,27 @@ object UserService {
     new UserService(userInfo, dataSource, googleServicesDAO, gpsDAO, gpsGroupSyncTopic, notificationDAO, samDAO, projectOwnerGrantableRoles)
 
   case class OverwriteGroupMembers(groupRef: RawlsGroupRef, memberList: RawlsGroupMemberList)
+
+  def getGoogleProjectOwnerGroupEmail(samDAO: SamDAO, project: RawlsBillingProject)(implicit ec: ExecutionContext): Future[RawlsGroupEmail] = {
```

Comment:
I completely agree. Since Brad is going to need this functionality soon, and restructuring may take some thought, I'm inclined to let it slide and fix in a follow-up. I do think that at least identifying the duplication in code (even via static functions) is a step in the right direction.
Follow-up tickets:
https://broadinstitute.atlassian.net/browse/GAWB-3667
https://broadinstitute.atlassian.net/browse/GAWB-3666